modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,704 @@
+
+For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to Tiny Comments].
+
+
+=== Merge tags 1.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Merge tags** premium plugin.
+
+**Merge Tags** 1.1.0 includes one addition, one fix, and one change:
+
+==== New mceMergeTag command to wrap selected content in a merge tag
+
+A new command, `mceMergeTag`, is now available as a Text Patterns option.
+
+Previously only a predefined list of merge tags could be used. When the new `mceMergeTag` command is added to `text_patterns`, any text selection content can be converted into a merge tag.
+
+As well, configuring the inline text_pattern with this command allows merge tags to be entered by manual typing.
+
+Type the prefix, some content, the suffix, and then a word boundary character, and the `mceMergeTag` command will immediately wrap that content in a complete merge tag.
+
+In the following example, the Merge Tag prefix and suffix are the default, `{{` and `}}`
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  plugins: "mergetags",
+  text_patterns: [
+    { start: '{{', end: '}}', cmd: 'mceMergeTag' }
+  ],
+  mergetags_list: [
+    {
+      value: 'merge.tag',
+      title: 'A merge tag.'
+    },
+  ],
+  toolbar: "mergetags",
+});
+----
+
+In this example, typing `{{string}}` followed by a word boundary character (such as a space, full-stop, question mark or comma) will result in the _string_ being turned into an active merge tag.
+
+==== Merge tags cannot exceed 255 characters in length
+
+In {productname} 6.3 a new character limit range of 0-255 was introduced.
+
+With this change, all merge tags are now subject to this limit including those predefined in the `mergetags_list` option.
+
+In {productname} 6.3, to support this feature, the editor will check all merge tag lengths to ensure they do not exceed 255 characters. 
+
+In the event the merge tag length exceeds this, the item will not be accepted by the editor.
+
+NOTE: Some non-ASCII characters or Unicode codepoints count as more than one character.
+
+For information on the **Merge tags** plugin, see: xref:mergetags.adoc[Merge Tags].
+
+
+=== Spellchecker Pro 3.1.1
+
+The {productname} 6.3 release includes an accompanying release of the **Spellchecker Pro** premium plugin.
+
+**Spellchecker Pro** 3.1.1 includes one fix:
+
+==== Spellchecker Pro could search outside of the {productname} editor for a language match.
+
+For spelling, a word’s language was identified by finding the nearest `lang` attribute anywhere in an ancestor HTML element to the word (ie, any elements that contain the word). 
+
+When {productname} was running in Inline mode, that included HTML ancestors that were provided by the host Browser, which are outside the {productname} editor instance.
```

Comment:
I think "outside page" is probably a bit more accurate than "host browser"

Reply:
More reflection, and, consequently, another edit.